### DISCUSSION TITLE
[Project Beta] Filter by issue ID

### DISCUSSION THREAD

**Discussion by marybeihl** [community/community#9931] [Projects and Issues]
In the "old" projects,  on the board, we could simply type the Issue ID in the filter field and the board will filter to just that issue

We cannot seem to figure out how to do that with this new beta projects.  

We realize that this could bring back multiple issues due to the project spanning repos but that's fine with us.

We use this feature a LOT so would really like to see it added back

**Comment by ronnyroeller** [comment]
Agreed. This is really painful, as we finally got the team to talk about issues by their ticket number...

**Reply by iangabrielsanchez** [reply]
It would be really useful to have this feature
